google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -707,6 +707,60 @@ public void testCreateTableWithRangePartitioning() {
     }
   }
 
+  @Test
+  public void testJsonType() throws InterruptedException {
+    String tableName = "test_create_table_jsontype";
+    TableId tableId = TableId.of(DATASET, tableName);
+    Schema schema = Schema.of(Field.of("jsonField", StandardSQLTypeName.JSON));
+    StandardTableDefinition standardTableDefinition = StandardTableDefinition.of(schema);
+    try {
+      // Create a table with a JSON column
+      Table createdTable = bigquery.create(TableInfo.of(tableId, standardTableDefinition));
+      assertNotNull(createdTable);
+
+      // Insert data into JSON column
+      Map<String, Object> jsonRow = new HashMap<>();
+      jsonRow.put("jsonField", "{\"class\" : {\"students\" : [{\"name\" : \"Jane\"}]}}");
+      InsertAllRequest request = InsertAllRequest.newBuilder(tableId).addRow(jsonRow).build();
+      InsertAllResponse response = bigquery.insertAll(request);
+      assertFalse(response.hasErrors());
+      assertEquals(0, response.getInsertErrors().size());
+
+      // Query the JSON column with string positional query parameter
+      String sql =
+          "SELECT JSON_VALUE(jsonField, \"$.class.students[0].name\") FROM "
```

Comment:
The projection could then change to SELECT jsonField.class.student.id where JSON_VALUE(jsonField, \"$.class.student.name\") = ? "

id should be 10.
It may also be great to use the INSERT syntax to add the columns above with JSONObjects.

Reply:
Done.